### ISSUE TITLE
keep old nightly builds around for a day

### ISSUE THREAD

**Issue by mikeshulman** [HoTT/book#846]
As discussed at #799, since the saunders redirect only updates once a day.

**Comment by mikeshulman** [comment]
Anyone have the travis-fu to do this?

**Comment by JasonGross** [comment]
I don't have a computer right now, but what you want to do is modify the following block in `etc/ci/update-nightlies.sh` (or whatever the filename is):
```bash
git rm -rf nightly || true
rm -rf nightly || true

mkdir nightly

git add -f $PDFS || exit 1

git mv $PDFS nightly/ || exit 1
```
so that rather than removing all of `nightly`, it only removes things in `$PDFS` (alternatively, check if `git mv` supports a `-f` flag to replace existing files), and things older than a day.  Stackoverflow can probably give you a `find` command to find/remove things older than a day.  (Also, you'll need to replace `mkdir nightly` with `mkdir -p nightly`, or a test for existence, so that it doesn't fail if `nightly` exists.)

**Comment by mikeshulman** [comment]
Ok, I gave it a try.

**Comment by mikeshulman** [comment]
#880

**Comment by peterlefanulumsdaine** [comment]
Closed by b5e98e8.
